packages/core/components/EditMetadata/NewAnnotationPathway.tsx

```diff
@@ -18,21 +31,87 @@ enum EditStep {
 
 interface NewAnnotationProps {
     onDismiss: () => void;
-    onUnsavedChanges: () => void;
+    hasUnsavedChanges: (arg: boolean) => void;
     selectedFileCount: number;
 }
 
+// Simplified version of status message
+interface AnnotationStatus {
+    status: ProcessStatus;
+    message: string | undefined;
+}
+
 /**
  * Component for submitting a new annotation
  * and then entering values for the selected files
  */
 export default function NewAnnotationPathway(props: NewAnnotationProps) {
+    const dispatch = useDispatch();
+    // Destructure to prevent unnecessary useEffect triggers
+    const { onDismiss, hasUnsavedChanges } = props;
+
     const [step, setStep] = React.useState<EditStep>(EditStep.CREATE_FIELD);
-    const [newValues, setNewValues] = React.useState<string | undefined>();
+    const [newValues, setNewValues] = React.useState<AnnotationValue | undefined>();
     const [newFieldName, setNewFieldName] = React.useState<string>("");
+    const [newFieldDescription, setNewFieldDescription] = React.useState<string>("");
     const [newFieldDataType, setNewFieldDataType] = React.useState<AnnotationType | undefined>();
```

Comment:
Should we just set a default for this if we `onSubmit` do anyway?

Reply:
added a default in [73a8316](https://github.com/AllenInstitute/biofile-finder/pull/406/commits/73a83161edfb1645a108729511b49fe509ca46e7)